RecoEgamma/EgammaElectronProducers/interface/LowPtGsfElectronSeedHeavyObjectCache.h

```diff
@@ -25,18 +25,18 @@ namespace lowptgsfeleseed {
     float trk_chi2red_ = -1.;
     float rho_ = -1.;
     float ktf_ecal_cluster_e_ = -1.;
-    float ktf_ecal_cluster_deta_ = -1.;
-    float ktf_ecal_cluster_dphi_ = -1.;
+    float ktf_ecal_cluster_deta_ = -42.;
```

Comment:
This is intentional. It was chosen as a default value as part of the training to ensure good separation from typical values when no track-cluster match is found. To change this would require a retraining for the BDT models.

Reply:
ok